test/test_crawl.py

```diff
@@ -1,18 +1,17 @@
-# type:ignore
-# As this file is no longer maintained, mypy shouldn't check this
-import os
+"""Runs a short test crawl.
```

Comment:
Check out https://github.com/mozilla/OpenWPM/issues/197. I disabled this test a while back because I don't love the idea of connecting to real websites during the tests.

Is there a way to restrict this test to only run on CI and not when a user runs the test suite locally?

Reply:
Sure there is, I added this

```python
@pytest.mark.skipif(
    "CI" not in os.environ or os.environ["CI"] == "false",
    reason="Makes remote connections",
)
```